dev/drivers/scripts/plots/cam/jevs_cam_radar_nbrcnt_plots_last90days.sh

```diff
@@ -0,0 +1,79 @@
+#PBS -N jevs_cam_radar_nbrcnt_plots_last90days_00
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=1:50:00
+#PBS -l place=vscatter:exclhost,select=1:ncpus=128
+#PBS -l debug=true
+#PBS -V
+
+
+set -x
+
+cd $PBS_O_WORKDIR
+
+
+############################################################
+# Load modules
+############################################################
+
+
+export model=evs
+export NET=evs
+export COMPONENT=cam
+export STEP=plots
+export RUN=atmos
+
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+source $HOMEevs/versions/run.ver
+module reset
+module load prod_envir/${prod_envir_ver}
+
+source $HOMEevs/dev/modulefiles/$COMPONENT/${COMPONENT}_${STEP}.sh
+evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
+
+
+############################################################
+# For dev testing
+############################################################
+export envir=prod
+export DATAROOT=/lfs/h2/emc/stmp/${USER}/evs_test/$envir/tmp
+export VERIF_CASE=radar
+export MODELNAME=${COMPONENT}
+export job=${PBS_JOBNAME:-jevs_${MODELNAME}_${VERIF_CASE}_${LINE_TYPE}_${STEP}}
+export jobid=$job.${PBS_JOBID:-$$}
+export COMIN=/lfs/h2/emc/vpppg/noscrub/${USER}/$NET/$evs_ver_2d
+export COMOUT=/lfs/h2/emc/vpppg/noscrub/${USER}/$NET/$evs_ver_2d/$STEP/$COMPONENT
+export nproc=64
+############################################################
+
+export vhr=${vhr:-${vhr}}
+export EVAL_PERIOD=${EVAL_PERIOD:-LAST90DAYS}
+export LINE_TYPE=${LINE_TYPE:-nbrcnt}
+
+export SENDMAIL=${SENDMAIL:-YES}
+export SENDCOM=${SENDCOM:-YES}
+export SENDECF=${SENDECF:-YES}
+export SENDDBN=${SENDDBN:-NO}
+export KEEPDATA=${KEEPDATA:-YES}
+
+export MAILTO=${MAILTO:-'marcel.caron@noaa.gov,alicia.bentley@noaa.gov'}
```

Comment:
Should we change the MAILTO to use andrew.benjamin@noaa.gov for all of the drivers?

Reply:
I'd agree with that.  I'll make the change and if @AliciaBentley-NOAA or @AndrewBenjamin-NOAA disagree I can revert.